aioimmich/jobs/__init__.py

```diff
@@ -0,0 +1,42 @@
+"""aioimmich jobs api."""
+
+from ..api import ImmichSubApi
+from .models import (
+    ImmichJobsStatus,
+    JobCommand,
+    JobStatus,
+)
+
+
+class ImmichJobs(ImmichSubApi):
+    """Immich jobs api."""
+
+    async def async_get_all_jobs_status(self) -> ImmichJobsStatus:
+        """Get all jobs status.
+
+        Returns:
+            all jobs status as `ImmichJobsStatus`
+        """
+        result = await self.api.async_do_request("jobs/status")
```

Comment:
`jobs/status` seems to be the wrong endpoint - [getAllJobsStatus](https://immich.app/docs/api/get-all-jobs-status) is at `jobs`

Reply:
Fixed!